sdk-setup/src/mb2

```diff
@@ -341,6 +342,66 @@ COMMANDS
         and "package" commands unless the '--all' option or '--rsync' method is
         used.
 
+    undeploy {--pkcon|--rpm|--rsync|--zypper} [--all] [-n|--dry-run]
+             [--] [[+|-]<pattern>...]
+        Undeploy build results previously deployed to a device.
+
+        Initially, device packages are collected based on the build host name
+        recorded in their metadata.  Just those packages that seem to
+        originate from the host where mb2 is running will be considered for
+        removal from the device.  For that it is vital that the host name is
+        stable and unique among the host names recorded by packages installed
+        from other sources.
+
+        Further, unless the '--all' option is used, just the build results of
+        the project under current working directory will be considered for
+        removal from the device.  When the '--all' option is used, any build
+        results originated from this build host will be considered.  (Note how
+        the effect of '--all' differs between 'deploy' and 'undeploy'.)
+
+        Package origin is determined based on the build host name recorded in
+        its metadata.  For that it is vital that 1) the build host name has
+        not changed since the time the packages were built and that 2) the
+        build host name is unique among the host names recorded by packages
+        installed from other sources.  See the 'engine' command to learn more
+        about the build host name.
+
+        With the '--dry-run' option, the list of (sub)packages that would be
+        removed may be previewed without applying the changes.
+
+        If needed, the selection can be furthermore adjusted with glob
+        <pattern>s, each of which may be preceded with '+' or '-' to opt for
+        either inclusion (default) or exclusion respectively as the desired
+        effect.  Later specified patterns may override earlier ones.
+        Extended, Bash-compatible patterns are allowed.  Options must precede
+        patterns if any.
+
+        Except for the '--rsync' method, the selected method is not required to
+        match the method previously used with the 'deploy' command.
+
+        The following methods of undeployment are available, corresponding to
+        the mandatory option:
+
+        --pkcon
+            Uses 'pkcon' to remove RPM package(s).  Requires the "Allow
+            untrusted software" option enabled in Settings, does not require
+            root access to the device.
+
+        --rpm
+            Uses plain 'rpm' to remove RPM package(s).  Requires root access
+            to the device.
+
+        --rsync
+            This method is special.  It does not work with RPM packages.  It
```

Comment:
I think the mention about being special is not interesting here as well. "This method does not work with RPM packages" would be sufficient.

Reply:
ditto